test-suite/src/test/require-jdk8/org/scalajs/testsuite/javalib/time/DurationTest.scala

```diff
@@ -0,0 +1,887 @@
+package org.scalajs.testsuite.javalib.time
+
+import java.time.temporal.{ChronoUnit, UnsupportedTemporalTypeException}
+import java.time.{LocalTime, DateTimeException, Duration}
+
+import org.scalajs.jasminetest.JasmineTest
+import org.scalajs.testsuite.javalib.ExpectExceptions
+
+object DurationTest extends JasmineTest with ExpectExceptions {
+  import Duration._
+  import ChronoUnit._
+
+  val dmin = Duration.ofSeconds(Long.MinValue)
+  val dmax = Duration.ofSeconds(Long.MaxValue, 999999999)
+
+  describe("java.time.Duration") {
+    it("should respond to `get`") {
+      val d0 = ofSeconds(0, 1)
+
+      expect(dmin.get(SECONDS)).toEqual(Long.MinValue)
+      expect(dmin.get(NANOS)).toEqual(0)
+      expect(ZERO.get(SECONDS)).toEqual(0)
+      expect(ZERO.get(NANOS)).toEqual(0)
+      expect(d0.get(SECONDS)).toEqual(0)
+      expect(d0.get(NANOS)).toEqual(1)
+      expect(dmax.get(SECONDS)).toEqual(Long.MaxValue)
```

Comment:
Oh. You may not use `.toEqual` with `Long`s. You have to write it like this:
```scala
expect(dmax.get(SECONDS) == Long.MaxValue).toBeTruthy
```
otherwise both the lhs and rhs are converted to Doubles to be given to the Jasmine test framework.